packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -179,39 +182,57 @@ it('Create a directly funded channel between two wallets ', async () => {
   expect(getChannelResultFor(channelId, resultA1.channelResults)).toMatchObject({
     status: 'running',
     turnNum: 3,
+    fundingStatus: 'Funded',
   });
 
   const resultB1 = await b.pushMessage(getPayloadFor(participantB.participantId, postFundA.outbox));
   expect(getChannelResultFor(channelId, resultB1.channelResults)).toMatchObject({
     status: 'running',
     turnNum: 3,
+    // TODO: the fundingStatus is incorrect as the funding table is not joined with channels table
+    //  when processing the pushMessage above
+    // fundingStatus: 'Funded',
```

Comment:
I doubt there is much of a performance hit — the table id indexed on `channelId`.